portal/src/presentation/markdownRenderer.ts

```diff
@@ -16,8 +16,8 @@ interface LinkProps extends Props {
 
 const defaultRenderer = {
     paragraph: ({ children }: Props) => createElement("p", { className: "jkl-p jkl-spacing--bottom-2" }, children),
-    list: ({ children }: Props) => createElement("ul", { className: "jkl-bullet-list jkl-p" }, children),
-    listItem: ({ children }: Props) => createElement("li", { className: "jkl-bullet-list__item" }, children),
+    list: ({ children }: Props) => createElement("ul", { className: "jkl-list jkl-p" }, children),
```

Comment:
Denne må endres til å ha med `jkl-list--button` i `className`, hvis ikke du endrer stilarket som diskutert over. Legg gjerne til funksjonalitet for å rendre nummererte lister her også hvis du føler deg kallet! 😊